focus-android/app/src/main/java/org/mozilla/focus/settings/advanced/SecretSettingsFragment.kt

```diff
@@ -39,6 +43,24 @@ class SecretSettingsFragment :
         }
     }
 
+    private fun setupSERPTelemetryServer() {
```

Comment:
Contrary to GL's point, let's stay consistent with acronym cases in code. Would still be good to capitalize it in comments and strings though
```suggestion
    private fun setupSerpTelemetryServer() {
```